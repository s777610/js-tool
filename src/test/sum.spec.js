const { sum } = require('./sum');

const num1 = 1;
const num2 = 2;

describe('unit test', () => {
  it('test 1', () => {
    const res = sum({ num1, num2 });
    expect(res).toMatchSnapshot();
  });
});
