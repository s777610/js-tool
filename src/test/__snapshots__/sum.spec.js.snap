// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unit test test 1 1`] = `3`;
